### ISSUE TITLE
Restrict wiki editing to collaborators?

### ISSUE THREAD

**Issue by Purpzie** [ju1ce/April-Tag-VR-FullBody-Tracker#145]
Hi! I was testing a wiki in another repository but accidentally had this one open. It actually allowed me to add a footer despite not being a contributor. Is that unintended?

**Comment by ju1ce** [comment]
Yes, wiki is open for contribution to everyone. I don't think there is any need to restrict it, I can always review any changes done.
